play/src/components/GameSessionSwitch.tsx

```diff
@@ -87,7 +87,7 @@ export default function GameSessionSwitch({
         <GameInProgress
           {...gameSession}
           apiClient={apiClient}
```

Comment:
Note `localModel` here is different from the depricated `localAnswer`. `localModel` contains the information stored in `localStorage` and now includes a `BackendAnswer`. A bit weird in terms of naming but I feel that it remains accurate (also includes stuff like which avatar a user selected)